src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+//import edu.wpi.first.wpilibj2.*;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+
+
+/** Add your docs here. */
+public class IntakeSubsystem extends SubsystemBase {
+  // Put methods for controlling this subsystem
+  // here. Call these from Commands.
+  private CANSparkMax m_motorController1 = new CANSparkMax(Constants.kIntakeMotorOneID, CANSparkMaxLowLevel.MotorType.kBrushless);
+  private CANSparkMax m_motorController2 = new CANSparkMax(Constants.kIntakeMotorTwoID, CANSparkMaxLowLevel.MotorType.kBrushless);
+  
+
+  public void IntakeController(){
+  
+  }
+  
+  public void IntakeSwitch(boolean on){
+    if (on){
+      m_motorController1.set(Constants.kIntakeMotorSpeed);
```

Comment:
Don't forget about the [follow function](https://codedocs.revrobotics.com/java/com/revrobotics/cansparkmax#follow(com.revrobotics.CANSparkMax)). This can be used to make one CANSparkMax always do the same thing as another. If this code becomes more complex it can save you a lot of copy/pasting